src/appleseed/renderer/modeling/light/spotlight.cpp

```diff
@@ -286,7 +291,8 @@ DictionaryArray SpotLightFactory::get_input_metadata() const
             .insert("min_value", "-360.0")
             .insert("max_value", "360.0")
             .insert("use", "optional")
-            .insert("default", "0.0"));
+            .insert("default", "0.0")
+            .insert("help", "TODO: document me"));
```

Comment:
"Rotate the spot light around its axis; only useful when using the light intensity is textured (gobo)"